GitAssignment/src/com/company/Main.java

```diff
@@ -3,11 +3,21 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+
+
     }
 
 
-    public static void checkSort(){
-        System.out.println("hello world");
+    public static boolean checkSort(double[] array){
```

Comment:
I'd suggest renaming the function to `isAscSorted` or something that indicates which kind of checking you're doing (your code is checking if it "**ascendingly**" sorted). It's just a naming convention, but the code is great. 
Good job.

Reply:
@bavly-adel You have pushed a file named `"GitAssignment/.idea/workspace.xml"` with a conflict with the master branch. Please resolve the conflict in order to merge it with master. 
**Hint**
The conflict is in an unimportant file (it is a configuration file used by your IDE), you can copy the content of that file from the **master** branch and then **push again** your files.